bitnami/jaeger/templates/migrate-job.yaml

```diff
@@ -0,0 +1,112 @@
+        app.kubernetes.io/component: migration
+        {{- if .Values.migration.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.migration.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+      {{- if .Values.migration.podAnnotations }}
+      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.migration.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+    spec:
+      restartPolicy: OnFailure
+      {{- if .Values.migration.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.migration.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      initContainers:
+        - name: jaeger-cassandra-schema-grabber
+          image: {{ include "jaeger.image" . }}
+          command: [ "cp", "-T", "-r", "/opt/bitnami/jaeger/cassandra-schema", "/cassandra-schema/" ]
+          volumeMounts:
+            - name: cassandra-schema
+              mountPath: "/cassandra-schema"
+      containers:
+        - name: jaeger-cassandra-migrator
+          image: {{ include "jaeger.cqlshImage" . }}
+          command:
+            - /bin/bash
+          args:
+            - -ec
+            - |
+              #!/bin/bash
+
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+
+              . /opt/bitnami/scripts/libos.sh
+
+              check_cassandra_connection() {
+                  echo "SELECT 1" | cqlsh -u $CASSANDRA_USERNAME -p $CASSANDRA_PASSWORD -e "QUIT"
+              }
+
+
+              info "Connecting to the Cassandra instance $CQLSH_HOST:$CQLSH_PORT"
+              if ! retry_while "check_cassandra_connection" 12 30; then
+                error "Could not connect to the database server"
+                exit 1
+              else
+                info "Connection check success"
+                sed -i 's/cqlsh -e "show version"/cqlsh -e "show version" -u $CASSANDRA_USERNAME -p $CASSANDRA_PASSWORD /' /cassandra-schema/create.sh
+                /cassandra-schema/docker.sh
+              fi
+          env:
+            - name: CQLSH
+              value: /opt/bitnami/cassandra/bin/cqlsh
+            - name: CQLSH_HOST
+              value: {{ include "jaeger.cassandra.host" . }}
+            - name: CQLSH_PORT
+              value: {{ include "jaeger.cassandra.port" . }}
+            - name: CASSANDRA_USERNAME
+              value: {{ include "jaeger.cassandra.user" . }}
+            - name: DATACENTER
+              value: {{ include "jaeger.cassandra.datacenter" . }}
+            - name: KEYSPACE
+              value: {{ include "jaeger.cassandra.keyspace" . }}
+            - name: CASSANDRA_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "jaeger.cassandra.secretName" . }}
+                  key: {{ include "jaeger.cassandra.secretKey" . }}
+          {{- if or .Values.migration.extraEnvVarsCM .Values.migration.extraEnvVarsSecret }}
```

Comment:
Missing the `extraEnvVars` value here

Reply:
Done [d6f9cf5](https://github.com/bitnami/charts/pull/13480/commits/d6f9cf55c60f76acf8b9039f5bda12c01b992f55)